packages/core/admin/admin/src/components/NpsSurvey/tests/index.test.js

```diff
@@ -1,16 +1,10 @@
 import React from 'react';
 
-import { lightTheme, ThemeProvider } from '@strapi/design-system';
-import { render, screen, waitFor, fireEvent, act } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
+import { fireEvent } from '@testing-library/react';
+import { render, waitFor, act, server } from '@tests/utils';
 import { rest } from 'msw';
-import { setupServer } from 'msw/node';
-import { IntlProvider } from 'react-intl';
-import { QueryClient, QueryClientProvider } from 'react-query';
 
-import NpsSurvey from '..';
-
-const toggleNotification = jest.fn();
+import NpsSurvey from '../index';
```

Comment:
What benefits do you see doing this way over the way it is?

Reply:
I've always done it this way since it's less code and it doesn't increase the complexity of reading it, but I've just done a global search on the project and there are more `../index` than `../`, so I'm fine either way

Reply:
I'm gonna leave it, I _think_ if you change the file name if it's this way it'll auto update 🤞🏼